### ISSUE TITLE
versionInfo as an alternative to versionIRI?

### ISSUE THREAD

**Issue by dgarijo** [oeg-upm/fair_ontologies#57]
While it's desirable to have a versionIRI, some ontos just have version info and that's it.
Should we do a test specific to version info? Shall we pass half the test if there is versionInfo instead?

**Comment by coolharsh55** [comment]
No, it should still be a 0% passed test because giving versionInfo without versionIRI means you have indicated there is a version but not indicated where using the versionIRI. Giving 50% pass for just versionInfo means all ontologies should have both versionInfo (50%) and versionIRI (50%).

**Comment by dgarijo** [comment]
But in the end we aim at having a version available. Having information on the version may be better than having none.

**Comment by coolharsh55** [comment]
Ah - I misunderstood your comment, sorry. Then it should be 50% versionInfo + 50% versionIRI as both are recommended in WIDOCO best practices.

**Comment by dgarijo** [comment]
Maybe we can consider 100% versionIRI (ideal) + 50%(optional but desirable) versioninfo.
But what you suggest makes sense as well.
